package.json

```diff
@@ -36,21 +36,22 @@
   "dependencies": {
     "@rjhilgefort/export-dir": "^2.0.0",
     "lodash": "^4.17.21",
-    "ramda": "^0.27.1",
+    "ramda": "^0.28.0",
     "string-format": "^2.0.0",
     "stringify-object": "^3.3.0"
   },
   "devDependencies": {
-    "@losant/eslint-config-losant": "1.4.3",
+    "@losant/eslint-config-losant": "1.5.0",
     "docdash": "^1.2.0",
-    "expect": "^26.6.2",
-    "fs-extra": "^9.0.1",
+    "expect": "^29.3.1",
+    "fs-extra": "^10.1.0",
     "handlebars": "^4.7.7",
-    "husky": "^6.0.0",
-    "jest": "^26.6.3",
-    "jsdoc": "^3.6.6",
-    "lint-staged": "^11.0.0",
+    "husky": "^8.0.2",
+    "jest": "^29.3.1",
+    "jsdoc": "^4.0.0",
+    "lint-staged": "^13.0.3",
     "present": "^1.0.0",
+    "taffydb": "^2.7.3",
```

Comment:
I had to add this dev dep in order to build the docs because of this error 
```
FATAL: Unable to load template: Cannot find module 'taffydb'
Require stack:
- /Users/erin@losant.com/Documents/git/omnibelt/node_modules/docdash/publish.js
- /Users/erin@losant.com/Documents/git/omnibelt/node_modules/jsdoc/cli.js
- /Users/erin@losant.com/Documents/git/omnibelt/node_modules/jsdoc/jsdoc.js
```